projects/ion/src/lib/link/link.component.ts

```diff
@@ -0,0 +1,51 @@
+import { CommonModule, NgClass } from '@angular/common';
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  Output,
+  SimpleChanges,
+} from '@angular/core';
+
+import { IonIconComponent } from '../icon';
+import { IonLinkProps } from './types';
+
+@Component({
+  standalone: true,
+  imports: [CommonModule, IonIconComponent, NgClass],
+  selector: 'ion-link',
+  templateUrl: './link.component.html',
+  styleUrls: ['./link.component.scss'],
+})
+export class IonLinkComponent implements OnChanges {
+  @Input() label?: IonLinkProps['label'] = '';
```

Comment:
```suggestion
  @Input() label: IonLinkProps['label'] = '';
```
You can remove the `?` when you initialize prop with a value.